docs/get-started/activation.md

```diff
@@ -27,7 +27,7 @@ If you would like to use the Otomi Console (Web UI) and Otomi API (for code vali
 - Go to [Otomi Cloud](https://portal.otomi.cloud)
 - Create an account
 - Click on `Register Cluster`
-- Agree with the [EULA](../../product/eula/v1)
+- Agree with the [EULA](/product/eula/v1)
```

Comment:
does this also need .md?

Reply:
yes

Reply:
for some reason links from and to /product reject `.md`, but it seems to work even with `/` appended. I'll let it slide for the time being as it already costed too much time